src/sass/modules/_m-vet-nav.scss

```diff
@@ -52,6 +53,10 @@ body.va-pos-fixed {
     position: relative;
   }
 
+  .vetnav--submenu-expanded & {
+    height: 0;
+  }
+
```

Comment:
It maybe be better to put this in the parent class instead to avoid the sort-of-confusing higher-level selector inside a lower-level element, but I liked the way this was nested and this way I didn't have to repeat the `#vetnav-menu` selector anywhere else.